pkg/operation/botanist/wanted_secrets.go

```diff
@@ -714,14 +715,14 @@ func (b *Botanist) generateWantedSecretConfigs(basicAuthAPIServer *secrets.Basic
 
 			// Secret definition for vpn-seed (OpenVPN client side)
 			&secrets.CertificateSecretConfig{
-				Name:       "vpn-seed",
+				Name:       kubeapiserver.SecretNameVPNSeedTLSAuth,
```

Comment:
```suggestion
				Name:       kubeapiserver.SecretNameVPNSeed,
```
Otherwise you get a "missing information" error.